src/basic/dirent-util.c

```diff
@@ -101,3 +100,14 @@ struct dirent *readdir_no_dot(DIR *d) {
                         return de;
         }
 }
+
+ssize_t posix_getdents(int fd, void *buf, size_t nbyte, int flags) {
+        assert(fd >= 0);
+        assert(buf);
+        assert(nbyte > 0);
+
+        if (flags != 0)
+                return -EINVAL; /* Currently flags must be zero. */
+
+        return getdents64(fd, buf, nbyte);
+}
```

Comment:
please move this to missing_libc or so if you touch this. I don't think a wrapper like this belongs in *-util

Reply:
Turns out moving this to the .c file is not required, so I'm just going to drop this change.